app/serializers/logs_serializer.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class LogsSerializer < ApplicationSerializer
+  attributes :id, :item_type, :event, :user_id, :username, :object, :created_at
```

Comment:
```suggestion
  attributes :id, :item_type, :event, :user_id, :username, :created_at
```
do not serialize the whole object here. specify what attributes you require.